src/CodeFormatter/CommandLineParser.cs

```diff
@@ -32,6 +32,18 @@ public sealed class CommandLineOptions
             verbose: false,
             useAnalyzers: false);
 
+        public static readonly CommandLineOptions ListAnalyzers = new CommandLineOptions(
```

Comment:
A new "canned" set of command line options to support the newly added /analyzers option. This lists the available analyzers, by analogy to /rules, which lists the available rules. Actually what it really lists is the available diagnostics, so maybe it should be "/diagnostics" instead. Let me know if you prefer that.